src/ui/views/Board/components/Board/Board.svelte

```diff
@@ -32,41 +41,106 @@
     columns = e.detail.items;
     onSortColumns(columns.map((col) => col.id));
   }
+
+  function createColumnMenu(column: Column) {
+    const menu = new Menu();
+
+    menu.addItem((item) => {
+      item
+        .setTitle(column.collapse ? "Expand column" : "Collapse column")
+        .setIcon(
+          column.collapse ? "chevrons-left-right" : "chevrons-right-left"
+        )
+        .onClick(() => {
+          onColumnCollapse(column.id, column.collapse);
+        });
+    });
+
+    if (column.id !== $i18n.t("views.board.no-status")) {
+      menu.addSeparator();
+
+      menu.addItem((item) => {
+        item
+          .setTitle($i18n.t("components.board.column.delete"))
+          .setIcon("trash-2")
+          .setWarning(true)
+          .onClick(() => {
+            onColumnDelete(
+              columns.map((col) => col.id),
+              column.id,
+              column.records
+            );
+          });
+      });
+    }
+
+    return menu;
+  }
 </script>
 
-<section
-  class="projects--board"
-  style={`grid-template-columns: repeat(${columns.length}, ${columnWidth}px);`}
-  use:dndzone={{
-    type: "columns",
-    items: columns,
-    flipDurationMs,
-    dropTargetStyle: {
-      outline: "none",
-    },
-  }}
-  on:consider={handleDndConsider}
-  on:finalize={handleDndFinalize}
->
-  {#each columns as column (column.id)}
-    <BoardColumn
-      {readonly}
-      {richText}
-      name={column.id}
-      records={column.records}
-      {onRecordClick}
-      onRecordAdd={() => onRecordAdd(column.id)}
-      onDrop={(record, records, trigger) => {
-        switch (trigger) {
-          case "droppedIntoZone":
-            onRecordUpdate(record, { ...column, records }, "addToColumn");
-            break;
-          case "droppedIntoAnother":
-            onRecordUpdate(record, { ...column, records }, "removeFromColumn");
-            break;
-        }
+<div>
+  <section
+    class="projects--board"
+    use:dndzone={{
+      type: "columns",
+      items: columns,
+      flipDurationMs,
+      dropTargetStyle: {
+        outline: "none",
+      },
+    }}
+    on:consider={handleDndConsider}
+    on:finalize={handleDndFinalize}
+  >
+    {#each columns as column (column.id)}
+      <div class="projects--board--column--dndwrapper">
```

Comment:
Should wrap the dnd child, as the style of the first dnd child is strictly managed by the dnd lib.
https://github.com/isaacHagoel/svelte-dnd-action/issues/284